src/main/java/seedu/duke/User.java

```diff
@@ -1,4 +1,154 @@
 package seedu.duke;
 
+import java.util.Scanner;
+
+/**
+ * User class keeps track of user's personal information.
+ */
 public class User {
+    public static String name;
+    public static Integer age;
+    public static Double height;
+    public static Double weight;
+    public static String gender;
+    public Scanner in = new Scanner(System.in);
+
+    public User() {
+        setup();
+    }
+
+    private void setup() {
+        System.out.println("Hi there, before we begin, let me get to know you :)");
+        System.out.println("Please enter your name:");
+        setName();
+        System.out.println("Please enter your age:");
+        setupAge();
+        System.out.println("Please enter your height (in m):");
+        setupHeight();
+        System.out.println("Please enter your weight (in kg):");
+        setupWeight();
+        System.out.println("Please enter your gender (Enter '1' for Male or '2' for Female):");
+        setupGender();
```

Comment:
Once we have Ui class we can consider moving these lines of code there instead, and call the required methods. Same for `viewProfile()`.